packages/sdk/src/swap/v2/types.ts

```diff
@@ -63,12 +62,13 @@ interface SwapChunk {
   latestSwapRescheduledAt: number | undefined;
   latestSwapRescheduledBlockIndex: number | undefined;
   fees: SwapFee[];
+  isDca: boolean;
```

Comment:
already have this on the swap - do we need it on the chunk too?

Reply:
yeah because if you access swap the type is `SwapFields | SwapChunk | (SwapFields & SwapChunk) | undefined` so it's a bit annoying not being able to access the `isDca` field in all cases